bindings/pyroot/cppyy/CPyCppyy/src/CPyCppyyModule.cxx

```diff
@@ -208,6 +208,12 @@ static PyTypeObject PyDefault_t_Type = {
 
 namespace {
 
+#if defined(__GNUC__)
+#pragma GCC diagnostic push
+// More recent versions of python change the second argument from a ssize_t to
```

Comment:
Thank you very much for the initiative! I would rather see some preprocessor checks based on `PY_VERSION_HEX` though :) Then we don't have to mess with the diagnostics.